app/pages/clinicworkspace/ClinicPatients.js

```diff
@@ -531,7 +531,8 @@ export const ClinicPatients = (props) => {
     [clinicBgUnits]
   );
 
-  const [activeFilters, setActiveFilters] = useLocalStorage('activePatientFilters', defaultFilterState, true);
+  const activeFiltersStorageKey = `activePatientFilters/${loggedInUserId}/${selectedClinicId}`;
```

Comment:
Question for sanity check - a "clinic" and a "workspace" are functionally the same thing in our platform, correct? The requirements are that the filters are stored `per user, per workspace`. This seems to work with that assumption in mind, but I just want to triple check.

Reply:
Yes, that's correct.